File: problema 7/fortran/problema_7.f90
program findEvenNumbers
    implicit none
    integer :: matriz(5,5)
    integer :: i, j

    ! Input elements of the matrix
    print *, "Ingrese los elementos de la matriz (5x5):"
    do i = 1, 5
        do j = 1, 5
            read *, matriz(i, j)
        end do
    end do

    ! Display the matrix and find even numbers
    print *, "Matriz ingresada:"
    do i = 1, 5
        do j = 1, 5
            write(*, '(I3)', advance='no') matriz(i, j)
            if (mod(matriz(i, j), 2) == 0) then
                print *, " (Par en posicion [", i, "][", j, "])"
            endif
        end do
        print *, ""
    end do

end program findEvenNumbers

File: problema 7/c-languaje/problema_7.c
#include <stdio.h>

int main() {
    int matriz[5][5];
    int i, j;

    // Input elements of the matrix
    printf("Ingrese los elementos de la matriz (5x5):\n");
    for (i = 0; i < 5; i++) {
        for (j = 0; j < 5; j++) {
            scanf("%d", &matriz[i][j]);
        }
    }

    // Display the matrix and find even numbers
    printf("Matriz ingresada:\n");
    for (i = 0; i < 5; i++) {
        for (j = 0; j < 5; j++) {
            printf("%d\t", matriz[i][j]);
            if (matriz[i][j] % 2 == 0) {
                printf("(Par en posicion [%d][%d])\n", i, j);
            }
        }
        printf("\n");
    }

    return 0;
}